SLHCUpgradeSimulations/Configuration/python/customise_TkRecoMaterial.py

```diff
@@ -0,0 +1,20 @@
+'''Customization functions for cmsDriver to change the phase-2 tracker reco material for geometry D49'''
+import FWCore.ParameterSet.Config as cms
+def customizeRecoMaterial(process):
```

Comment:
will the new file eventually become the default, so this customize function will only be needed temporarily?

Reply:
@kpedro88 yes, eventually, if validation at https://its.cern.ch/jira/projects/PDMVRELVALS/issues/PDMVRELVALS-79 is satisfactory.